website/static/js/comment.js

```diff
@@ -63,6 +103,54 @@ var exclusifyGroup = function() {
     }
 };
 
+var convertMentionHtmlToMarkdown = function(commentContent) {
+    var content = commentContent || '';
+    var pattern = '<span[^>]*?data-atwho-guid="([a-z\\d]{5})"[^>]*?>((@|\\+)[\\w\\s]+)<\/span>',
+        regex = new RegExp(pattern),
```

Comment:
BTW, it's ok to diverge from the style guidelines iff this is copy-n-pasted code; just make clear that it was copied.